tests/test_authentication.py

```diff
@@ -94,6 +105,14 @@ def test_unknown_creds(self):
 
         eq_(exc.exception.detail, 'authentication failed')
 
+    def test_alternative_credential_lookup(self):
+        sender = self._sender(credentials=ALTERNATIVE_CREDS)
+        req = self._request(sender)
+        lookup_path="tests.test_authentication.alternative_lookup"
```

Comment:
Just a nit: I try to follow [pep8](http://docs.python-guide.org/en/latest/writing/style/) where there would be spaces between the assignment operator like `lookup_path = ...`